### DISCUSSION TITLE
2.0 => 2.1 upgrade: no spark charts on the sites page

### DISCUSSION THREAD

**Discussion by kleuter** [plausible/analytics#4220] [Self-Hosted Support] [answered]
Upgrade went fine but the sites page is missing graphs (site names are blurred)

![image](https://github.com/plausible/analytics/assets/2600624/a2df744a-2bff-4320-a035-54353e1fa8b4)

![image](https://github.com/plausible/analytics/assets/2600624/ad93f970-fdaf-4879-87ee-76826d96e299)

any ideas? thanks.

**Comment by ruslandoga** [comment, marked as answer]
👋 @kleuter 

Please make sure your reverse proxy allows WebSocket connections.

**Comment by kleuter** [comment]
Thank you, solved. This should have been mentioned in the 2.0->2.1 upgrade I think.
